client/landing/stepper/declarative-flow/internals/steps-repository/videomaker-setup/index.tsx

```diff
@@ -0,0 +1,109 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+
+import { Button } from '@automattic/components';
+import { StepContainer } from '@automattic/onboarding';
+import { useDispatch, useSelect } from '@wordpress/data';
+import { useI18n } from '@wordpress/react-i18n';
+import classNames from 'classnames';
+import React, { FormEvent } from 'react';
+import FormattedHeader from 'calypso/components/formatted-header';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { ONBOARD_STORE } from '../../../../stores';
+import type { Step } from '../../types';
```

Comment:
Could/should these be referenced in the other direction?

ex: `calypso/landing/stepper/stores`
ex: `calypso/landing/stepper/declarative-flow/internals/types`

Reply:
Ah good call I think my editor did that: Fixed in 399432e.